### PR TITLE
Fix integration test 'PageOpen and PageClose actions'

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Comment by timvandermeij** [comment]
This integration test sadly fails pretty much all the time now, but only on Linux. Should we perhaps disable it on Windows only for the time being given that this PR doesn't seem to solve the issue? Note that there are more integration tests that can fail, but those are more intermittent.

**Comment by calixteman** [comment]
This PR likely fix an issue: I saw an exception when I ran the test in the vm.
And a strange fact is that I ran successfully the test several times in the vm.
The failure is likely due to an un-executed PageOpen callback and I really wonder why.
My feeling is that there is a bug somewhere (not in the test) and we should figure out it.
We can disable the test but I think it's a good way to forget it.

**Comment by timvandermeij** [comment]
Oh, definitely. If the issue in this test is indeed composed of multiple issues, fixing one of them is already great! In that case we can indeed try this to at least hopefully decrease the frequency of the issue.

File: test/integration/scripting_spec.js [hunk 1]
```diff
@@ -534,10 +534,15 @@ describe("Interaction", () => {
                 page,
                 getSelector(refOpen),
                 async () => {
+                  const sel = getSelector(refOpen);
+                  await page.waitForSelector(sel, {
+                    timeout: 0,
```
Comment (timvandermeij) [inline test/integration/scripting_spec.js:539]: Is this safe in the sense that if this selector would not trigger we would basically be waiting here forever? Or is there a global timeout on the Jasmine tests or so?

**Comment by timvandermeij** [comment]
If the comment above is answered, I'm fine with merging this if it fixes an exception you encountered, but in that case I'd propose we reword the commit message to `Fix exception in PageOpen and PageClose actions integration test ` since it otherwise implies that the entire test is fixed which isn't the case yet.

**Comment by timvandermeij** [comment]
@calixteman Ping for the above so we can hopefully get this landed to at least improve the situation for this failing test.

**Comment by calixteman** [comment]
I'll take the time "asap" to fix this stuff.

**Review by naxl — APPROVED** [review]

**Comment by timvandermeij** [comment]
@calixteman Gentle ping for this; if it's still useful it'd be nice to get this merged given the many integration test failures we have.

**Pull request closed without merging**